lib/bots/githubbot.ts

```diff
@@ -61,7 +62,7 @@ export class GithubBot extends ProcBot.ProcBot<string> {
         // The `github` module is a bit behind the preview API. We may have to override
         // some of the methods here (PR review comments for a start).
         this.githubApi = new GithubApi({
-            // debug: true,
+            //debug: true,
```

Comment:
Nit: Comment should have a space after `//`, perhaps this one shouldn't be there at all though?